### PR TITLE
docs: flexible workspace timeouts

### LINKED ISSUE #3267: [Docs][Pricing page] Update flexible workspace timeouts
### Description

See [PR15815](https://github.com/gitpod-io/gitpod/pull/15815). Previously, workspace timeout could be extended to 180 minutes, now, it can be extended to an arbitrary number.

Information needs to be updated in 2 places:

[Pricing page feature table](https://www.gitpod.io/pricing)
Screenshot 2023-01-18 at 16 52 36

Suggestion:
- Change hover-over information to: "Manually extend workspace timeout as long as you want"

[Docs](https://www.gitpod.io/docs/introduction/learn-gitpod/the-life-of-a-workspace#timeouts)
Screenshot 2023-01-18 at 17 00 11

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description

Related https://github.com/gitpod-io/gitpod/pull/15815

**Do Not Merge Until the above PR is deployed** _(ETA: tomorrow)_ 

## Related Issue(s)

Fixes #3267 

## How to test

Open Preview, Read & Verify the following:

- [`/pricing`](https://deploy-preview-3268--gitpod-kumquat.netlify.app/pricing)

image

- [`/docs/introduction/learn-gitpod/the-life-of-a-workspace#timeouts`](https://deploy-preview-3268--gitpod-kumquat.netlify.app/docs/introduction/learn-gitpod/the-life-of-a-workspace#timeouts)
- [`/docs/references/gitpod-cli#timeout`](https://deploy-preview-3268--gitpod-kumquat.netlify.app/docs/references/gitpod-cli#timeout)

## Release Notes

```release-note
docs: Flexible Workspace Timeouts
```

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3268"></a>

### CODE REVIEW

File: src/lib/contents/pricing.ts [hunk 1]
```diff
@@ -670,7 +670,7 @@ export const pricingTableToc: FeatureTableToc[] = [
     type: "tooltip",
     data: {
       text: "Timeout boost",
-      tooltip: "Manually extent workspace timeout to 180 minutes",
+      tooltip: "Manually extent workspace timeout as long as you want.",
```
Comment (MichaelAring) [inline src/lib/contents/pricing.ts:673]:
```suggestion
      tooltip: "Manually extend workspace timeout as long as you want.",
```

**Review by MichaelAring — CHANGES_REQUESTED** [review]

**Comment by MichaelAring** [comment]
Tagging @burningion: this is not a meaningful change in the docs, but still letting you know in case you want to improve the content.

File: gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ This ensures that you are starting from a clean slate with proper configuration.
 ## Timeouts
 
 Any running workspace will automatically stop after some time of inactivity. Normally, this timeout is 30 minutes but is extended to **60 minutes if you have an _Unleashed_ plan**.
-Furthermore, _Unleashed_ users can manually boost the timeout of a workspace to 180 minutes. This comes in handy, e.g. in case you want to go out for a longer lunch or meeting and don't like restarting your workspace when coming back.
+Furthermore, _Unleashed_ users can manually boost the timeout of a workspace to as long as you want ([learn more](/docs/references/gitpod-cli#timeout)). This comes in handy, e.g. in case you want to go out for a longer lunch or meeting and don't like restarting your workspace when coming back.
```
Comment (burningion) [inline gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md:20]: What is `Unleashed`?  Do we mean paid users? And isn't there a 24 hour limit?

File: gitpod/docs/references/gitpod-cli.md [hunk 3]
```diff
@@ -228,6 +238,14 @@ Shows the current workspace's timeout.
 gp timeout show
 ```
 
+### extend
+
+Extends the current workspace's timeout.
```
Comment (burningion) [inline gitpod/docs/references/gitpod-cli.md:243]: by how much?

**Review by burningion — CHANGES_REQUESTED** [review]
Added a few comments

File: gitpod/docs/references/gitpod-cli.md (hunk 3 above)
Comment (Siddhant-K-code) [reply gitpod/docs/references/gitpod-cli.md:243]: This is an unrelated change through this PR (just re-ordered the sequence). But, I will take a look at this. Thanks

File: gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md (hunk 2 above)
Comment (Siddhant-K-code) [reply gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md:20]: Again, It's an unrelated change that should be addressed at https://github.com/gitpod-io/website/pull/2895. I will take care of this.

File: src/lib/contents/pricing.ts (hunk 1 above)
Comment (Siddhant-K-code) [reply src/lib/contents/pricing.ts:673]: This is being discussed internally [[1](https://gitpod.slack.com/archives/C02EN94AEPL/p1674059258956459)], [[2](https://gitpod.slack.com/archives/C046Z4BBP2N/p1674061475535929)]. Let's Hold this PR Until we got the final heads up.

File: gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md [hunk 4]
```diff
@@ -16,12 +16,12 @@ This ensures that you are starting from a clean slate with proper configuration.
 
 ## Timeouts
 
-Any running workspace will automatically stop after some time of inactivity. Normally, this timeout is 30 minutes but is extended to **60 minutes if you have an _Unleashed_ plan**.
-Furthermore, _Unleashed_ users can manually boost the timeout of a workspace to 180 minutes. This comes in handy, e.g. in case you want to go out for a longer lunch or meeting and don't like restarting your workspace when coming back.
+Any running workspace will automatically stop after some time of inactivity. Normally, this timeout is 30 minutes but is extended to **60 minutes if you have an** [**_Paid plan_**](/pricing).
```
Comment (burningion) [inline gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md:19]: "but is extended to 60 minutes if you have a paid plan."

File: gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md [hunk 5]
```diff
@@ -16,12 +16,12 @@ This ensures that you are starting from a clean slate with proper configuration.
...
+Furthermore, _Paid_ users can manually boost the timeout of a workspace upto 24 hours ([learn more](/docs/references/gitpod-cli#timeout)). This comes in handy, e.g. in case you want to go out for a longer lunch or meeting and don't like restarting your workspace when coming back.
```
Comment (burningion) [inline gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md:20]: "up to"

File: gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md [hunk 6]
```diff
@@ -16,12 +16,12 @@ This ensures that you are starting from a clean slate with proper configuration.
...
 
 To do that, open the editor's Command Palette (<Keybind>CtrlCmd + Shift + P</Keybind>) and search for "Gitpod: Extend Workspace Timeout".
 
-**Note**: If you do not have an _Unleashed_ plan, this command is not available.
+**Note**: If you do not have an _Paid_ plan, this command is not available.
```
Comment (burningion) [inline gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md:24]: why are we capitalizing Paid?

File: src/lib/contents/pricing.ts [hunk 7]
```diff
@@ -670,7 +670,7 @@ export const pricingTableToc: FeatureTableToc[] = [
     type: "tooltip",
     data: {
       text: "Timeout boost",
-      tooltip: "Manually extent workspace timeout to 180 minutes",
+      tooltip: "Manually extend workspace timeout upto 24 hours.",
```
Comment (burningion) [inline src/lib/contents/pricing.ts:673]: "up to" (missing a space)

**Review by burningion — CHANGES_REQUESTED** [review]

File: gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md (hunk 6 above)
Comment (Siddhant-K-code) [reply gitpod/docs/introduction/learn-gitpod/the-life-of-a-workspace.md:24]: for using [this](https://www.gitpod.io/pricing) abbr.:

image

**Comment by Siddhant-K-code** [comment]
Also, Related Bug Report => https://github.com/gitpod-io/gitpod/issues/15945

File: gitpod/docs/references/gitpod-cli.md [hunk 8]
```diff
@@ -203,21 +203,30 @@ gp tasks stop --all
...
+gp timeout show
 ```
 
-### extend
-
-Extends the current workspace's timeout.
-
 > **Note:** You can only have one workspace with extended timeout at a time.
 
 The default timeout, and the ability to extend a workspace timeout depends on your [plan](https://gitpod.io/plans) or [team plan](https://gitpod.io/teams).
```
Comment (jldec) [inline gitpod/docs/references/gitpod-cli.md:211]: This line is no longer relevant - please replace with

`The default timeout, and the ability to extend a workspace timeout depends on your [billing configuration](/docs/configure/billing)`

(b.t.w. relative links allow the preview links to stay on site)

**Review by jldec — APPROVED** [review]
LGTM with one change requested.

File: gitpod/docs/references/gitpod-cli.md (hunk 8 above)
Comment (Siddhant-K-code) [reply gitpod/docs/references/gitpod-cli.md]: Resolved ✅

**Pull request merged**